src/Interactive/vbi/My Project/Settings.Designer.vb

```diff
@@ -1,7 +1,7 @@
-'------------------------------------------------------------------------------
```

Comment:
I'm *pretty* sure the project system/compiler don't care about it.  It's just generated by the settings designer when you click on the Project Properties page.